packages/node/src/indexer/indexer.manager.ts

```diff
@@ -123,53 +121,45 @@ export class IndexerManager {
     this.storeService.setTransaction(tx);
 
     let poiBlockHash: Uint8Array;
-    if (this.project.network.type === 'substrate') {
-      try {
-        const isUpgraded = block.specVersion !== this.prevSpecVersion;
-        // if parentBlockHash injected, which means we need to check runtime upgrade
-        const apiAt = await this.apiService.getPatchedApi(
-          block.block.hash,
-          block.block.header.number.unwrap().toNumber(),
-          isUpgraded ? block.block.header.parentHash : undefined,
-        );
+    try {
+      // if parentBlockHash injected, which means we need to check runtime upgrade
```

Comment:
That comment was about the apiAt = getPatchedApi.
You get rid of this part (for now or totally ?), so you can also remove the comment